tunes/Delvin House.abc

```diff
@@ -0,0 +1,12 @@
+X: 1
+T: Delvin House
+R: Strathspey
+M: C
+L: 1/16
+K: C
+r:16
+C: Trad
+|: e2 | c4 G3e c4 G3E | Dd3d3c BG3A3B | c3ed3e c3eA3c | G3-AG3-F Gc3 c4 :| 
+e2 |: {^f}g3ec3e cdef g3e | f3dB3d Bcde e3d |1 g3-ec3-e g3ec3G |
+Af3 e3d e3cc3e :|2 eg3 de3 ce3 Ac3| G3-AG3F Gc3 c2 ||
+Z: Mackin, from a cute old book of hundreds of tunes from her Violin teacher's Grandmother. 
```

Comment:
a line at the end of a file is good practice @MorrisNerd2

Reply:
iirc one of these libraries we use for the book creation is also able to lint abc files which gives the option of checking the files in gh actions as well

Reply:
I created a separate issue for an on PR change.